odoo/fields.py

```diff
@@ -2792,6 +2795,93 @@ def _record_ids_per_res_model(self, records):
         return model_ids
 
 
+class Command(enum.IntEnum):
+    """
+    :class:`~odoo.fields.One2many` and :class:`~odoo.fields.Many2many` fields
+    expect a special command to manipulate the relation they implement.
+    Internally, each command is a tuple where the first element is a mandatory
+    integer that identifies the command, the two next elements are arguments
```

Comment:
"to next" -> "next two". Also if applicable the second element is an `id` (otherwise it's generally though not necessarily `0` or `False`).